services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: int = None, actors_ids: int = None) -> str:
```

Comment:
The function `get_movies` is annotated to return a `str`, but it actually returns a `QuerySet` of `Movie` objects. Consider changing the return type annotation to `QuerySet[Movie]`.